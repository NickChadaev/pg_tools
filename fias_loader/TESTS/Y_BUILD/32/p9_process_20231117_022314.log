2023-11-17 02:23:14 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_32 postgres /home/n.chadaev@abrr.local/Y_BUILD/32/stage_gar_c_32.csv 2023-11-13
--------------------------------------------------------
[02:23:14] ... Брянская обл., id_region: 33, Execute: /home/n.chadaev@abrr.local/Y_BUILD/32/stage_gar_c_32.csv, process_id:25345
[02:23:14] ... Start process
[02:23:14] ...  -- Version
[02:23:14] ...  -- Очистка от данных
[02:23:14] ...  -- Убираем индексы
[02:23:14] ...  -- Set UNLOGGED
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[02:23:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_REESTR_OBJECTS_20231113_c21eb4dd-9b35-4d62-8fa8-fa5781be69ee.XML
[02:34:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_ADDR_OBJ_20231113_be0bfb48-0413-4bdd-bc30-c123bb824a2b.XML
[02:34:38] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_HOUSES_20231113_a3c725f9-be14-4101-962e-6a6784446fb2.XML
[02:55:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_STEADS_20231113_02da5c4a-9f2d-471c-8a6a-f370e20dd846.XML
[02:56:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_APARTMENTS_20231113_738bfe51-f1eb-482b-8016-28dbd5434ce4.XML
[03:01:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_ROOMS_20231113_ef8d203e-125c-4b3e-b572-91af4905330d.XML
[03:01:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_CARPLACES_20231113_fba0200d-5359-4784-9533-df3a256a83cb.XML
[03:01:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_ADM_HIERARCHY_20231113_b1b9319a-d8e7-4357-abea-2e5b43ed1e46.XML
[03:14:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_MUN_HIERARCHY_20231113_169c46be-f465-4495-909b-ab7e16c01ca6.XML
[03:27:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_CHANGE_HISTORY_20231113_e6607be8-7c3f-4cac-898b-4f906aa119bb.XML
[04:16:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_ADDR_OBJ_DIVISION_20231113_d907c117-ad27-4062-aa05-4fdf6c0f4b0a.XML
[04:16:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_ADDR_OBJ_PARAMS_20231113_06b44ff8-8e2c-4d66-8f4b-6766275535df.XML
[04:20:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_STEADS_PARAMS_20231113_8f1e625d-5d83-4f0d-b896-b8fbc286d8d6.XML
[04:42:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_HOUSES_PARAMS_20231113_a27e5e01-755a-4341-a299-08e32cdf5398.XML
[06:09:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_APARTMENTS_PARAMS_20231113_3285ae82-22e1-486d-a18c-ed80a017c4bb.XML
[06:18:23] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_ROOMS_PARAMS_20231113_820af58e-150f-46a0-8fdc-a15bf4e8aff0.XML
[06:18:24] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_CARPLACES_PARAMS_20231113_e4759d61-003b-4240-a5ef-1b5c6c4ecb7c.XML
[06:18:24] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/32/AS_NORMATIVE_DOCS_20231113_49a0a76a-2be5-4c6d-be8c-eced0ba100a0.XML
[06:20:12] ...  -- Data finish
[06:20:12] ...  -- Set LOGGED
[06:20:46] ... Stop process
--------------------------------------------------------
