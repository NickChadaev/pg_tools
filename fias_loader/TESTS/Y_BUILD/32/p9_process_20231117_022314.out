 save_gar_version 
------------------
               82
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                82
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 935272
gar_fias.as_addr_obj: 22431
gar_fias.as_houses: 1543639
gar_fias.as_steads: 132016
gar_fias.as_apartments: 371173
gar_fias.as_rooms: 1517
gar_fias.as_carplaces: 158
gar_fias.as_adm_hierarchy: 1002652
gar_fias.as_mun_hierarchy: 1058092
gar_fias.as_change_history: 4120493
gar_fias.as_addr_obj_division: 36
gar_fias.as_addr_obj_params: 375568
gar_fias.as_steads_params: 1760153
gar_fias.as_houses_params: 7053415
gar_fias.as_apartments_params: 683905
gar_fias.as_rooms_params: 1511
gar_fias.as_carplaces_params: 2
gar_fias.as_normative_docs: 139781
